scripts/dependency/requirements.Linux.external.txt

```diff
@@ -42,7 +42,7 @@ azure-mgmt-msi==0.2.0
 azure-mgmt-network==2.0.0rc3
 azure-mgmt-nspkg==2.0.0
 azure-mgmt-policyinsights==0.1.0
-azure-mgmt-rdbms==1.2.0
+azure-mgmt-rdbms==1.3.0
```

Comment:
You do not need to manually update these two files. But it is ok, you don't need to revert it, either.